### DISCUSSION TITLE
main.exe not launching

### DISCUSSION THREAD

**Discussion by mxoliveira** [Khopa/dcs_liberation#2074] [Q&A]
I get the new 5.1.0 version, but it doesn`t lauch... I`m running with administration rights on windows 10... Anyone got this issue?

**Comment by Starfire13** [comment]
Check that your antivirus isn't blocking it.

**Reply by mxoliveira** [reply]
I use windows defender and all ready configured in the exclusions programs.

**Reply by bazacop** [reply]
yep same problem for me i get Fatal error come up saying failed to execute script main and i have this in the log
2022-03-12 23:29:33,040 :: INFO :: DCS Liberation 5.1.0
2022-03-12 23:29:33,040 :: DEBUG :: Python version 3.9.9 (tags/v3.9.9:ccb0e6a, Nov 15 2021, 18:08:50) [MSC v.1929 64 bit (AMD64)]
2022-03-12 23:29:33,153 :: ERROR :: Using default theme because liberation_theme.json does not exist
2022-03-12 23:29:33,154 :: INFO :: Loading stylesheet: style-dcs.css
2022-03-12 23:29:33,154 :: DEBUG :: Loading Liberation preferences from C:\Users\bazza\AppData\Local\DCSLiberation\liberation_preferences.json 

i have also seen a post in here from earlier 4.1 i believe with same problem but had no answer to it. mind only just started happening a day ago really strange as was fine before

**Reply by mxoliveira** [reply]
The version 4.1.1 works fine but the 5.0.0 or 5.1.0 continues not running

**Reply by bazacop** [reply]
i think it might be something to do with a windows update or something like that as i said mine was working fine and the only thing i can think thats change its windows updated. might be totally wrong

**Reply by Starfire13** [reply]
If it were due to a Windows update error, lots of folks would be getting this issue. However, 99+% of people can run Liberation just fine, which I think rules out Windows update. I would suggest reinstalling Liberation as you might have accidentally deleted or misplaced certain files. Make sure you are not installing it in any of the program files folders. Make sure you have set up your paths properly during installation.

**Reply by bazacop** [reply]
well im at a Loss as i have deleted run ccleaner made sure its gone to correct place with is user\ name\ saved games and still not loading up and log says what it states above very strange any idea what any of that log means as i have no clue especially this bit
2022-03-12 23:29:33,153 :: ERROR :: Using default theme because liberation_theme.json does not exist

**Reply by mxoliveira** [reply]
Continues not working here too

**Reply by bazacop** [reply]
even tried to see if registry was messed up but that seems to be all good

**Reply by Starfire13** [reply]
> well im at a Loss as i have deleted run ccleaner made sure its gone to correct place with is user\ name\ saved games and still not loading up and log says what it states above very strange any idea what any of that log means as i have no clue especially this bit
> 
> 2022-03-12 23:29:33,153 :: ERROR :: Using default theme because liberation_theme.json does not exist
> 
> 

That error isn't the problem as there is no theme. Everyone gets that error. Can you delete your log file, attempt to run Liberation, then attach the log file here? I'd like to see the log file from just a single attempted run.

**Reply by bazacop** [reply]
no worries here you go
[liberation.log](https://github.com/dcs-liberation/dcs_liberation/files/8264940/liberation.log)

**Reply by Starfire13** [reply]
> no worries here you go
> 
> [liberation.log](https://github.com/dcs-liberation/dcs_liberation/files/8264940/liberation.log)
> 
> 
> 
> 

There's no error there, so it's not that Liberation is exiting with an error. It just isn't being allowed to run. It's probably something specific to your system.

**Reply by bazacop** [reply]
ok thanks for your help i will have to look deeper into my pc something might have broke. like i said at start was working great then stopped running all of a sudden. hopefully will get it sorted probably  by luck  lol hope i do i was enjoying that. and great work btw :)

**Reply by Raskil** [reply]
Please check that you have installed Liberation into a folder, where the hole path including that folder does not contain non-asciii characters.

**Reply by bazacop** [reply]
im sorrry but i dont understand what you mean by non-asciii  characters

**Reply by Raskil** [reply]
Make sure that dcs is put in a path that contains only "A-Z,a-z,0-9,_,-" as letters.

Correct: "C:\Users\raskil\dcs-liberation"
Wrong: "C:\Users\Möller\dcs-liberation" note the ö in the user name... that is a non ascii character.

**Reply by bazacop** [reply]
oh ok,
 No all looks good in the path
